.github/workflows/test.yml

```diff
@@ -17,7 +17,7 @@ jobs:
     - name: Download Memgraph Docker image
       if: steps.cache-memgraph-docker.outputs.cache-hit != 'true'
       env:
-        MG_VERSION: '1.3.0'
+        MG_VERSION: '1.4.0'
```

Comment:
move env above steps and use the MG_VERSION everywhere where a version is mentioned. In your case, check the cache step, you use 1.3.0 for the key while you changed to 1.4.0 here. 
Instead, you can write
```
key: cache-memgraph-v${{ env.MG_VERSION }}-docker-image
```